swaglyrics/spotify.py

```diff
@@ -1,4 +1,6 @@
 import platform
+searched_song = None
```

Comment:
why is this needed?

Reply:
This is unnecessary, sorry about this, I was trying to make a function for that infinite `while` loop in order to make a test case for that and cover that code part.